pkg/rpcclient/invoker/invoker.go

```diff
@@ -212,14 +212,25 @@ func iterateNext(rpc RPCSessions, sessionID uuid.UUID, iterator *result.Iterator
 		num = DefaultIteratorResultItems
 	}
 
-	if iterator.ID != nil {
```

Comment:
@roman-khimov I think I am still missing something about https://github.com/nspcc-dev/neo-go/issues/3272#issuecomment-1863207006
Previous struct had some invariants which were not violated, now we construct a new structure, extending possible invariants (mostly to have the same interface in both `unwrap.*` functions.

Reply:
Ok, now return array and iterator as-is.